mylist.rb

```diff
@@ -1,3 +1,5 @@
+# Use methods for MyList
+
 require_relative Enumerable
```

Comment:
- [ ] Kindly consider fixing your import statement in the `mylist.rb`, e.g `require_relative 'enumerable'` so your enumerable works properly

Reply:
Done